app/api/connectors/package.scala

```diff
@@ -16,13 +16,11 @@
 
 package api
 
-import api.models.errors.{DownstreamError, MtdError}
+import api.models.errors.DownstreamError
 import api.models.outcomes.ResponseWrapper
 
 package object connectors {
 
-  type MtdIdLookupOutcome = Either[MtdError, String]
```

Comment:
Why have we made this to remove the type and instead add it to the connector?

Reply:
Because the return type is tightly associated with the connector (and it now wraps a specific Error case class holding the status code rather than an MtdError).